src/exodus/wallet.cpp

```diff
@@ -0,0 +1,90 @@
+#include "wallet.h"
+#include "wallet/wallet.h"
+#include "wallet/walletdb.h"
+#include "walletdb.h"
+
+namespace exodus {
+
+SigmaPublicKey Wallet::CreateSigmaMint(
+    uint32_t propertyId,
+    uint8_t denomination)
+{
+    SigmaPrivateKey key;
+
+    do {
+        key.Generate();
+    } while (!key.IsValid());
+
+    SigmaEntry e;
+    e.SetNull();
```

Comment:
After `e` in constructed it should already null so we should not need to call `SetNull`.